api/src/main/java/org/pmiops/workbench/db/dao/UserService.java

```diff
@@ -287,14 +292,34 @@ public User createUser(
     return user;
   }
 
-  public User submitDataUseAgreement(Integer dataUseAgreementSignedVersion) {
+  public UserDataUseAgreement submitDataUseAgreement(
+      Integer dataUseAgreementSignedVersion, String initials) {
     final Timestamp timestamp = new Timestamp(clock.instant().toEpochMilli());
-    return updateUserWithRetries(
-        (user) -> {
-          user.setDataUseAgreementCompletionTime(timestamp);
-          user.setDataUseAgreementSignedVersion(dataUseAgreementSignedVersion);
-          return user;
+    UserDataUseAgreement dataUseAgreement = new UserDataUseAgreement();
+    dataUseAgreement.setDataUseAgreementSignedVersion(dataUseAgreementSignedVersion);
+    dataUseAgreement.setUserId(userProvider.get().getUserId());
+    dataUseAgreement.setUserFamilyName(userProvider.get().getFamilyName());
+    dataUseAgreement.setUserGivenName(userProvider.get().getGivenName());
+    dataUseAgreement.setUserInitials(initials);
```

Comment:
Please test with periods in initials, like "O.J.C". Also, with only 6 chars, I can't put spaces if I want.